### ISSUE TITLE
Mac Material Design color swatch tooltips are missing

### ISSUE THREAD

**Issue by bleroy** [xamarin/Xamarin.PropertyEditing#346]
> VS bug [#703876](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/703876)

**Comment by lewing** [comment]
As currently written the MaterialView control uses layers for all the color swatches.  My understanding is either the swatches need to be converted to NSViews and set the a tooltip or MaterialView needs to implement the area related tooltip protocol bits.
